reference/docs-conceptual/discover-powershell.md

```diff
@@ -278,96 +261,16 @@ Cmdlet          Stop-Process                 7.0.0.0    Microsoft.PowerShell.Man
 ```
 
-As you can see, knowing the type of a command, can greatly narrow down your search after commands
-that might be interesting for you.
-
-## Exercise - calling your first command
-
-In this exercise, you'll learn how to run your first command.
-
-1. Start a PowerShell console by typing `pwsh`:
-
-   ```powershell
-   pwsh
-   ```
-
-1. Run the following `$PSVersionTable.PSVersion`:
-
-   ```powershell
-   $PSVersionTable.PSVersion
-   ```
-
-   Your output looks similar to this:
-
-   ```output
-   Major  Minor  Patch  PreReleaseLabel BuildLabel
-   -----  -----  -----  --------------- ----------
-   7      1      0
-   ```
-
-   Congrats, you've successfully run your first command and was able to get information of what
-   version of PowerShell is installed on your system.
-
-## Exercise - find related commands
-
-In this exercise your goal is to learn more about a command. In doing so you'll also find things
-like what type it operates on and what other similar commands operate on that same type.
-
-1. Ensure you have a started PowerShell shell
-1. Run the command `Get-Process`:
-
-   ```powershell
-   Get-Process | Get-Member | Select-Object TypeName -Unique
-   ```
-
-   Your output resembles this:
-
-   ```output
-   TypeName
-   --------
-   System.Diagnostics.Process
-   --------
-   ```
-
-   What you're getting back, are the types returned by the command `Get-Command`. At this point you
-   can now find out what other command also operates on these types.
-
-1. Run the command `Get-Command`:
-
-   ```powershell
-   Get-Command -ParameterType Process
-   ```
-
-   Your output resembles this:
-
-   ```output
-   CommandType     Name                         Version    Source
-    -----------     ----                        -------    ------
-    Cmdlet          Debug-Process               7.0.0.0    Microsoft.PowerShell.Managem…
-    Cmdlet          Enter-PSHostProcess         7.1.0.0    Microsoft.PowerShell.Core
-    Cmdlet          Get-Process                 7.0.0.0    Microsoft.PowerShell.Managem…
-    Cmdlet          Get-PSHostProcessInfo       7.1.0.0    Microsoft.PowerShell.Core
-    Cmdlet          Stop-Process                7.0.0.0    Microsoft.PowerShell.Managem…
-    Cmdlet          Wait-Process                7.0.0.0    Microsoft.PowerShell.Managem…
-   ```
-
-   Congratulations, you've managed to find other commands that operates on the same type `Process`.
-   Using `Get-Member` is a good starting point to understand what other commands you should check
-   out next.
-
-## Summary
-
-In this first part, you learned what PowerShell is and what areas in can be used. You where then
-taught about cmdlets and in particular `Get-Command`, `Get-Verb` and `Get-Member`. Knowing theses
-cmdlets is important as it teaches you how to learn. In the next part, you'll learn how to use the
-powerful help system.
+As you can see, knowing the output type of a command can help narrow down your search for related
+commands.
```

Comment:
Nit: Suggest striking `As you can see` - if it's obvious to the reader, the phrase can be skipped. If it isn't obvious to them, it has a (mildly) negative connotation that they weren't clever enough to pick up on it. I see this as the same (although lesser) category of phrase as "You can easily..."

```suggestion
Knowing the output type of a command can help narrow down your search for related commands.
```